src/components/ChannelList/ChannelList.tsx

```diff
@@ -61,10 +60,36 @@ export type ChannelListProps<
   Me extends UnknownType = DefaultMessageType,
   Re extends UnknownType = DefaultReactionType,
   Us extends UnknownType = DefaultUserType
-> = Partial<ChannelsContextValue<At, Ch, Co, Ev, Me, Re, Us>> & {
+> = Partial<
```

Comment:
I don't know how much is on this context but looks like we are taking a lot of it, could we do an Omit with less?

Reply:
Prolly easier to read as what's in it than not so nvm